src/ui/MessageContent/index.scss

```diff
@@ -5,7 +5,6 @@
   display: inline-flex;
   flex-direction: row;
   width: 100%;
-  height: 100%;
```

Comment:
@HoonBaek for some odd reason, I had to remove this for profile container to work outside MessageContent component. Removing this does not affect backward compatibility.